### ISSUE TITLE
Tutorial?

### ISSUE THREAD

**Issue by GiorgosAlexakis** [nmi-lab/decolle-public#18]
Is there any tutorial notebook? It would be really helpful . Can I try another DVS dataset(with the same file format) easily or do I need to implement the conversion on my own? For example the action recognition dataset from here uses the aedat format. https://github.com/CrystalMiaoshu/PAFBenchmark

**Comment by weinman** [comment]
(This issue should probably go in the [torchneuromorphic](https://github.com/nmi-lab/torchneuromorphic/) repo, but anyway...) 


By way of analogy with the [DVSGestureDataset](https://github.com/nmi-lab/torchneuromorphic/blob/8a396917028b7fca091c91e8cad28ccebe729367/torchneuromorphic/dvs_gestures/dvsgestures_dataloaders.py#L35) It looks like you'd be able to create a subclass of NeuromorphicDataset and implement all the same methods in a very similar way.

It seems as though the repo you sent has data in a slightly different format, because the bitmasks it uses for the fields are narrower than those in the IBM DVS data (i.e., contrast [this](https://github.com/CrystalMiaoshu/PAFBenchmark/blob/67b7ee2ed1c8ac3c76a12d03af0f6c0e65925a64/snn.py#L41) with that given by IBM [here](https://research.ibm.com/dvsgesture/), and implemented [here](https://github.com/nmi-lab/torchneuromorphic/blob/8a396917028b7fca091c91e8cad28ccebe729367/torchneuromorphic/utils.py#L239).

Of course, the other thing you'll have to worry about is labeling the data streams if you're doing supervised learning. 

Long story short, there's a bit of conversion to do; it won't be turnkey, but you've got two templates to guide you from one (PAFBenchmark) to the other (DVSGestures in torchneuromorphic). Good luck!
